packages/cli/src/lib/builder/plugins.test.ts

```diff
@@ -111,23 +114,39 @@ describe('forwardFileImports', () => {
         throw new Error('options.external is not a function');
       }
 
-      expect(options.external('./my-module', '/dev/src/index.ts', false)).toBe(
-        false,
-      );
       expect(
-        options.external('./my-image.png', '/dev/src/dir/index.ts', false),
+        options.external(
+          './my-module',
+          resolvePath(mockDir.path, 'dev', 'src', 'index.ts'),
```

Comment:
```suggestion
          mockDir.resolve('dev/src/index.ts'),
```

same for these complex ones - it does read better

Reply:
Oh, missed that this was a thing. Definitely agree it reads better! Fixing!

Reply:
Changed all places where resolve or join was used in combination with `mockDir.path`. Much better, thanks for the suggestion!